docs/search/search_for_content.md

```diff
@@ -26,6 +26,11 @@ By selecting an option from a drop-down menu in the top right corner, you can so
 
 ![Sorting menu](img/sorting_menu.png)
 
+The search engine not only displays suggestions, but also supports spell checking.
+If there is a typo in your search phrase, the search engine suggests a related, accurate term.
```

Comment:
It's not only on typo it seems.

![basic_search.png](https://ez-systems-developer-documentation--290.com.readthedocs.build/projects/userguide/en/290/search/img/basic_search.png)
[basic_search.png](https://ez-systems-developer-documentation--290.com.readthedocs.build/projects/userguide/en/290/search/img/basic_search.png) also 🪄 "put a spell on user"

I don't know exactly why it was triggered in my search example but it worth a check.
(on low result count? by comparing terms' result counts?)

Reply:
If I understand you correctly - it also shows suggestions. Yes, it does, and we have it described: https://doc.ibexa.co/projects/userguide/en/latest/search/search_for_content/.